docs/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
```

Comment:
Did we add a section in the documentation repo to reference this newly added env vars?

Reply:
https://github.com/smartcontractkit/documentation/pull/983